### DISCUSSION TITLE
Installing and referring to data files using Poetry

### DISCUSSION THREAD

**Discussion by jbadwaik** [python-poetry/poetry#8861] [General] [answered]
I have a poetry project with a `myprog` script. 

Now, I want to install some files in `<install-prefix>/share/myprog/` and I want to access them while running `myprog`. However, I am coming up empty handed on how to handle this using poetry. I know I can install the files using the include directive in pyproject.toml. But now how to ensure that I can get the path to those files correctly?
importlib.resource doesn't help because it only looks relative to directory of current module. 

Is there any good way to approach this problem?

**Comment by dimbleby** [comment, marked as answer]
this does not seem to be anything to do with poetry.  What would the answer be if you had built your package using some other tool?

**Comment by jbadwaik** [comment]
Thanks for the  answer and links in the other issue. It seems that I had an different (possibly older) idea of what python packages are meant to achieve.
